docs/source/reference/rest_api.rst

```diff
@@ -1142,3 +1177,10 @@ Examples:
             http://localhost:5000/api/v4/nodes/a67fba41/links/outgoing?full_type="data.dict.Dict.|"
 
     would first search for the outgoing of the node with *uuid* starting with "a67fba41" and then select only those nodes of full_type *data.dict.Dict.|*.
+
+.. |Computer| replace:: :py:class:`Computer <aiida.orm.Computer>`
+.. |Node| replace:: :py:class:`Node <aiida.orm.Node>`
+.. |ProcessNode| replace:: :py:class:`ProcessNode <aiida.orm.ProcessNode>`
+.. |CalcJobNode| replace:: :py:class:`CalcJobNode <aiida.orm.CalcJobNode>`
+.. |User| replace:: :py:class:`User <aiida.orm.User>`
+.. |Group| replace:: :py:class:`Group <aiida.orm.Group>`
```

Comment:
I thought we had to specify the complete Python dot-separated path to the classes. And why not use the more easily readable :py:class:`~aiida.orm.users.User`? The tilde (`~`) will make sure only `User` is written in the text.

Reply:
Agreed on both counts - fixed!